Libraries/LibGC/RootVector.h

```diff
@@ -46,6 +46,12 @@ class RootVector final
 
     virtual ~RootVector() = default;
 
+    RootVector(Heap& heap, Span<T> const& other)
```

Comment:
This should probably be a `ReadonlySpan`. It's not an issue in this PR, but if `HTMLFormElement::reset_form()` were `const`, it wouldn't compile.
(We can also pass the span by value here)